pkg/interceptors/cel/cel.go

```diff
@@ -58,15 +59,15 @@ func (w *Interceptor) ExecuteTrigger(request *http.Request) (*http.Response, err
 		return nil, fmt.Errorf("error creating cel environment: %w", err)
 	}
 
-	body, err := request.GetBody()
-	if err != nil {
-		return nil, fmt.Errorf("error getting request body: %w", err)
-	}
-	defer body.Close()
-	payload, err := ioutil.ReadAll(body)
-	if err != nil {
-		return nil, fmt.Errorf("error reading request body: %w", err)
+	var payload = []byte(`{}`)
```

Comment:
I don't feel too strongly about this, but should we be consistent with the webhook handler and default to []byte{}?

Reply:
Sure, we'd have to do it the other way around though since `makeCelEnv` errors out if you pass it []byte{}